kubecost/templates/cluster-controller/cluster-controller-actions-bucket.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if (.Values.clusterController).enabled }}
-{{- if not (.Values.clusterController).storageConfigSecret }}
+{{- if (.Values.clusterController).storageConfig }}
```

Comment:
Updated this to only create the Secret if the `storageConfig` exists; on the previous condition, if neither `storageConfigSecret` **nor** `storageConfig` were to exist, this would still create the Secret but with nil contents